lib/stat_tracker.rb

```diff
@@ -54,5 +54,11 @@ def lowest_total_score
     output.total_game_score
   end
 
+  def average_goals_per_game
+    games_count = @games.count.to_f
+    sum_of_goals = (@games.map {|game| game.total_game_score}.to_a).sum
 
-end
+    sum_of_goals_divided_by_game_count = (sum_of_goals / games_count).round(2)
+    sum_of_goals_divided_by_game_count
```

Comment:
Hey Nico,

This looks great and all the test past on my end.  I guess my only question is, do we need to have `sum_of goals_divided_by_game_count` saved into a variable?

I believe this method will also pass if we simply  have 
```ruby
(sum_of_goals / games_count).round(2)
```

As the last line?

Reply:
done